tools/sleepy.py

```diff
@@ -56,7 +57,12 @@ def main():
   source_file_name = args.program  # type: str
   with open(source_file_name) as program_file:
     program = program_file.read()
-  ast = make_program_ast(program)
+  try:
+    ast = make_program_ast(program)
+  except CompilerError as ce:
+    print(str(ce))
```

Comment:
Okay, we can do it like this.
I feel like we should add a `--debug` flag though, which will then still print the full stack trace in case this is an compiler error (those do happen quite frequently still.. xD), and not actually an error in the program.